pandas/core/apply.py

```diff
@@ -303,13 +303,14 @@ def agg_list_like(self) -> DataFrame | Series:
         -------
         Result of aggregation.
         """
-        return self.agg_or_apply_list_like(op_name="agg")
+        kwargs = self.kwargs
+        return self.agg_or_apply_list_like(op_name="agg", **kwargs)
 
     def compute_list_like(
         self,
         op_name: Literal["agg", "apply"],
         selected_obj: Series | DataFrame,
-        kwargs: dict[str, Any],
+        **kwargs: dict[str, Any],
```

Comment:
Can you revert this change. If you make it `**kwargs`, then users will run into an error if they try to pass through e.g. `selected_obj` as a keyword argument.